[WIP] Support default arguments as additional signatures (#20)
wesselb/plum: README.md

## Keyword Arguments and Default Values

Default arguments can be used. The type annotation must match the default value otherwise 
an error is thrown. 

> Different default values can be used in different dispatch rules.

Reviewer: ```suggestion
As the example below illustrates, different default values can be used for different methods:
```

```python
from plum import dispatch

@dispatch

> def f(x:int, y:int=3):

Reviewer: ```suggestion
def f(x: int, y: int = 3):
```

(I think this is what Black prescribes.)

Author: yeah, ideally black could be run on readme.md... 
I'm very bad at enforcing style manually

    return f"Value for 2nd y: {y}"

@dispatch

> def f(x:float, y:float=3.0):

Reviewer: ```suggestion
def f(x: float, y: float = 3.0):
```

Idem.
